h2o-r/h2o-package/R/modelselection.R

```diff
@@ -284,7 +284,7 @@ h2o.modelselection <- function(x,
   model <- .h2o.modelJob('modelselection', parms, h2oRestApiVersion=3, verbose=FALSE)
   return(model)
 }
-.h2o.train_segments_modelselection <- function(x,
+.h2o.train_segments_modelSelection <- function(x,
```

Comment:
It looks a little bit weird to me now. Two styles mixed together. I will keep the old one or add underscore even it looks different than the method under...WDYT?

```suggestion
.h2o.train_segments_model_selection <- function(x,
```

Reply:
This happens automatically